tests/testthat/test-new-dataset.R

```diff
@@ -93,19 +93,47 @@ with_mock_crunch({
             '{"element":"shoji:entity","body":{"name":"helper.R"}}'
         )
     })
-    test_that("newDataset with a schema posts to sources", {
+    test_that("newDataset with an .sss schema posts to sources", {
         expect_POST(
-            newDataset(x = "helper.R", schema = "helper.R"),
+            newDataset(x = "helper.R", schema = "schema.sss"),
```

Comment:
I have a slight preference for doing this with a tempfile (`tempfile("schema", fileext=".sss")`) rather than making 2 new files to further clutter up the repository.

(Obvious counterpoint, there's already a ton of junk, but I think I'd rather stash it somewhere in the `inst/` folder if you think it should be permanent.)